content/sensu-go/5.14/guides/clustering.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "How to run a Sensu cluster"
+linkTitle: "Running a Sensu Cluster"
+description: "Clustering is important to make Sensu more highly available, reliable, and durable. It can help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents. Read the guide to configure a Sensu cluster."
+weight: 120
+version: "5.13"
+product: "Sensu Go"
+platformContent: false
+menu: 
+  sensu-go-5.13:
+    parent: guides
+---
+
+- [What is a Sensu cluster?](#what-is-a-sensu-cluster)
+- [Why use clustering?](#why-use-clustering)
+- [Configuring a cluster](#configuring-a-cluster)
+- [Adding sensu agents to the cluster](#adding-sensu-agents-to-the-cluster)
+- [Cluster health](#cluster-health)
+- [Managing cluster members](#add-a-cluster-member)
+- [Security](#security)
+  - [Client-to-server transport security with HTTPS](#client-to-server-with-https)
+  - [Client-to-server authentication with HTTPS client certificates](#client-to-server-auth-with-https)
+  - [Peer communication authentication with HTTPS client certificates](#peer-auth-https)
+  - [Sensu agent with HTTPS](#sensu-agent-https)
+- [Using an external etcd cluster](#using-an-external-etcd-cluster)
+- [Troubleshooting](#troubleshooting)
+
+## What is a Sensu cluster?
+
+A Sensu cluster is a group of [at least three][1] sensu-backend nodes, each connected to a shared etcd cluster, using Sensu's embedded etcd or an external etcd cluster. Creating a Sensu cluster ultimately configures an [etcd cluster][2].
+
+## Why use clustering?
+
+Clustering is important to make Sensu more highly available, reliable, and durable. It will help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents.
+
+_NOTE: We recommend using a load balancer to evenly distribute agent connections across the cluster._
+
+## Configuring a cluster
+
+The sensu-backend arguments for its store mirror the [etcd configuration flags][3], however the Sensu flags are prefixed with `etcd`. For more detailed descriptions of the different arguments, you can refer to the [etcd docs][4] or the Sensu [backend reference][15].
+
+You can configure a Sensu cluster in a couple different ways (we'll show you a few below) but it's recommended to adhere to some etcd cluster guidelines as well.
+
+> The recommended etcd cluster size is 3, 5 or 7, which is decided by the fault tolerance requirement. A 7-member cluster can provide enough fault tolerance in most cases. While a larger cluster provides better fault tolerance, the write performance reduces since data needs to be replicated to more machines. It is recommended to have an odd number of members in a cluster. Having an odd cluster size doesn't change the number needed for majority, but you gain a higher tolerance for failure by adding the extra member *(Core OS).*
```

Comment:
```suggestion
> The recommended etcd cluster size is 3, 5, or 7 members, and the size is decided by the fault tolerance requirement. A 7-member cluster can provide sufficient fault tolerance in most cases. A larger cluster provides better fault tolerance, but reduces the write performance because data needs to be replicated to more machines. We recommend an odd number of members in a cluster. An odd cluster size doesn't change the number needed for majority, but you gain a higher tolerance for failure by adding the extra member *(Core OS)*.
```